app/views/users/show.html.erb

```diff
@@ -151,6 +151,8 @@
       </button>
       <div id="user-profile-dropdownmenu" class="profile-dropdownmenu">
         <a href="/report-abuse?url=https://dev.to/<%= @user.username %>">Report Abuse</a>
+        <button id="user-profile-dropdownmenu-block-button" data-profile-user-id="<%= @user.id %>" class="block-button" type="button">Block</button>
```

Comment:
I think the text should maybe read `Block @<%= @user.username %> ` because that's how Twitter does it and it makes it clear what "block" means.

I think it makes sense for "block" to be above report abuse, as it seems like the more primary function here.